### DISCUSSION TITLE
Homekit “no response”

### DISCUSSION THREAD

**Discussion by iTzJacopoo** [mongoose-os-apps/shelly-homekit#1591] [Q&A]
Please i need help my 1plus doesn't work anymore on homekit.

**Comment by mappet2011** [comment]
poor problem description, try supply power off and on again, if it does not help once again but switch on / off 5x (the switch input) within 1 minute, then connect to shelly AP (shelly xxx wlan, go to 192.168.33.1) and set it up again properly as it was before, if it was a fixed ip then use same ip again, usually homekit will recognize it then.

your first time? this could happen more often, write it down

**Reply by iTzJacopoo** [reply]
I saw that my Apple TV gen1 was showing like an hub (witch it shouldn't?)
I don't know how to change that because gen1 shouldn't work like that.
If I remove my Apple TV from HomeKit it then works but i need both...
What can I do?

**Reply by mappet2011** [reply]
what did you change? you write that it does not work ANYMORE? you need at least one apple tv, homepod, .. as control.

**Reply by iTzJacopoo** [reply]
it worked in local for 2 months then one day it started conflicting with the apple tv that i always had in.
I knew that i needed an homepod or an apple tv (im getting an homepod for Christmas) to make it work remotely.
![image](https://github.com/user-attachments/assets/2f46eb9a-5ff6-463f-b0d4-f4f4ebb8ccbf)

**Comment by mappet2011** [comment]
so you have a homepod already as your main control and everything is fine, right? it stops working (no respond) when you do/did what? plug in apple tv gen 1? honestly i dont think that this will impact homekit as gen 1 does not support homekit. with other words homekit will not even recognize apple tv gen 1. so what do you mean with „remove from homekit“?

**Comment by iTzJacopoo** [comment]
no i don't have an homepod yet. I just have my apple tv 1.
im not home now, i will try to repair my apple tv.
Since when it showed like an hub my shellys do not work with homekit.
Im sure it is the apple tv's fault because when i changed the apple tv wifi to the one the shelly is connected it worked in remote too (it shouldn't 🤯)
ps: im sorry for my understanding and grammar but im italian and my first language is italian.

**Reply by mappet2011** [reply]
no problem 🙂, gen1 does not support homekit, it must be newer version then, is it **4k** gen1 perhaps? you could anyway check if there is an update on apple tv available and install it. ensure that everything is in the same wlan and restart every device. remove / connect the shelly again to homekit (new device) when the apple tv is connected as well.

**Reply by iTzJacopoo** [reply]
Look as I create a new home just to test my shelly and it works.
Then i add my Apple TV to it and it stops working, everything is updated and on the same wlan.![image](https://github.com/user-attachments/assets/c8c715da-fa00-4060-8e07-6825710f8b1d)![image](https://github.com/user-attachments/assets/0157391f-fcf4-4573-8309-18aac82ae4e5)

**Reply by iTzJacopoo** [reply]
oh... and this is a picture of my apple tv model and a picture of what is the problem (on my opinion)
PS: the apple tv was on when i took the screenshot.
![image](https://github.com/user-attachments/assets/5ff0151c-54c2-42c9-99bf-1e1d4769e2b1)
![image](https://github.com/user-attachments/assets/1165a7f6-130b-4a4d-96d7-94343c9db580)

**Reply by mappet2011** [reply]
this is tv 4k gen1 and this one is ready for homekit, so that is clear now. dont think that the problem is apple tv. try to remove the shelly and add it again in homekit as new device, means tap on it, go into settings, scroll to the bottom, remove it and then add it as new device in homekit. therefor find the ip (192.168.x.y) of the shelly and enter ip in browser, then you see at the top of the page (red) if the shelly needs an update (be careful, thiscould be tricky) and you can use the qr code to add it to homekit as new device.

**Reply by iTzJacopoo** [reply]
Apple says my model is not supported🧐
And even if it is, why does it says "no response" even if its on? (look at previous picture)
They are both updated to the last version.
And i tried multiple times to re-pair the shelly to homekit.

**Reply by mappet2011** [reply]
puh, frustrating, perhaps it is because of the transfer from local to remote handling, you have been in the homekit firmware? see pic
![IMG_0885](https://github.com/user-attachments/assets/fa7a3072-c93f-49f3-9726-a0b53e0703bd)

**Reply by iTzJacopoo** [reply]
either im on local or on remote its the same story.
may i contact you privately?
